src/Microsoft.TemplateEngine.Edge/TemplateUpdates/NupkgInstallUnitDescriptorFactory.cs

```diff
@@ -48,7 +48,7 @@ public bool TryCreateFromMountPoint(IMountPoint mountPoint, out IReadOnlyList<II
             descriptorList = allDescriptors;
 
             if (mountPoint.Info.Place != null
-                && Directory.Exists(mountPoint.Info.Place)  // nupkgs are expanded to files on install, so Directory.Exists() is more appropriate
+                && File.Exists(mountPoint.Info.Place)
```

Comment:
I think this exists check should run through IPhysicalFileSystem, shouldn't it?

Reply:
Yep, fixed.